src/realmGeol.ttl

```diff
@@ -1026,7 +1026,7 @@ soreag:Inlier rdf:type owl:Class ;
 
 ###  http://sweetontology.net/realmGeol/InnerCore
 soreag:InnerCore rdf:type owl:Class ;
-                 rdfs:subClassOf soreag:Mantle ;
+                 rdfs:subClassOf soreag:Core ;
```

Comment:
Seems like InnerCore and OuterCore are subclasses of GeosphereLayer, with a partOf Relation to Core, not a subclass relation.

Reply:
Not one of our mistakes- we haven't touched properties like this.

I actually do have another PR waiting in the wings for a tidy-up of the Solid Earth, Hydrosphere & Atmosphere and their layers (all using isPartOf relations) but I will wait for this PR to get through before lodging that one.